classlib/src/main/java/org/teavm/classlib/java/lang/TSystem.java

```diff
@@ -119,6 +119,10 @@ public static TString getProperty(TString key, TString def) {
         return value != null ? value : def;
     }
 
+    public static String setProperty(TString key, TString value) {
```

Comment:
I think I can provide fair implementation for getProperty/setProperty myself

Reply:
I intentionally decided to not allow `setProperty` in Bck2Brwsr. It influences global state and I wanted to prevent that.

Also `SecurityException` is better. At least the test knows that it shall not expect the functionality to work...

Reply:
One may agree or disagree, but designers of Java class library decided to include `getProperty`/`setProperty` methods into `System` class, and they are meant to provide some contract. Were they right or wrong, I choose to follow these rules, since TeaVM either does not support method at all, or supports it as close to OpenJDK, as possible

Reply:
I added support for system properties. Please, rebase to latest master.